spec/classes/profile/elastic/filebeat/configs/ulib_spec.rb

```diff
@@ -19,7 +19,7 @@
         it do
           is_expected.to contain_file(file)
             .that_notifies('Service[filebeat]')
-            .with_content(%r{^\s+document_type: ulib$})
+            .with_content(%r{^\s+ulib_type: 90_day$})
```

Comment:
An explanatory note for posterity: naming the type 90_day is a setup to allow for writing logs to different locations on disk based on potentially multiple retention policies for different types of logs or services (e.g. if Fulcrum and HT want longer than default retention. We likely won't be going any further with this plan though, as we are moving towards grafana alloy and loki.